bridge/slack/handlers.go

```diff
@@ -293,3 +302,12 @@ func (b *Bslack) fileIsAvailable(file *slack.File) bool {
 	}
 	return false
 }
+
+func (b *Bslack) fileIsRecentlyCached(file *slack.File) bool {
```

Comment:
@patcon the actual fix would be to fix the `fileIsAvailable` method just above that (for whatever reason) has it's comparisons inverted.

Reply:
Thanks. I was treading lightly on that as I wasn't clear if it was serving another purpose in handleDownloadFile. But I just confirmed that it was causing another bug (couldn't upload another file of the same name until it expired from cache), and so I just removed it from the download func.

Let me know if you think that's incorrect! Thanks for bearing with me, @Helcaraxan :)